fdbserver/include/fdbserver/MockGlobalState.h

```diff
@@ -82,6 +84,10 @@ class MockStorageServer : public IStorageMetricsService {
 	};
 
 	static constexpr uint64_t DEFAULT_DISK_SPACE = 1000LL * 1024 * 1024 * 1024;
+	// assume 80k operation will saturate the CPU linearly. We can change the model to a better one in the future.
+	static constexpr double DEFAULT_OP_CPU_MULTIPLIER = 100.0 / 80000;
```

Comment:
What's the benefit of declaring `DEFAULT_OP_CPU_MULTIPLIER`? It appears to only be used on the next line.

Reply:
This gives a default value, but the actual value used to calculate CPU can be changed by mock workload.
I didn't put it as a knob because I think it's better to set it through the workload parameters.